src/components/hosts/HostsTable.tsx

```diff
@@ -316,10 +319,15 @@ const HostsTable: React.FC<HostsTableProps> = ({ cluster, skipDisabled = false }
         <TableBody rowKey={rowKey} />
       </Table>
       <EventsModal
-        title={`Host Events: ${getHostInventory(showEventsModal)?.hostname || showEventsModal}`}
+        title={
```

Comment:
PF complains if the title is not set to the Modal:
```
index.js:1 Modal: Specify at least one of: title, aria-label, aria-labelledby. 
    at Modal (http://10.37.146.83:3000/static/js/0.chunk.js:9813:161)
    at EventsModal (http://10.37.146.83:3000/static/js/0.chunk.js:183947:23)
    at HostsTable (http://10.37.146.83:3000/static/js/0.chunk.js:185774:24)
    at BaremetalInventory (http://10.37.146.83:3000/static/js/0.chunk.js:186231:24)
```
Either move the `showEventsModal` condition inside the string or add aria-label prop to EventsModal